tests/test_bin.py

```diff
@@ -280,19 +280,19 @@ def test_bin_hy_no_main():
 @pytest.mark.parametrize('scenario', [
     "normal", "prevent_by_force", "prevent_by_env"])
 @pytest.mark.parametrize('cmd_fmt', [
-    'hy {fpath}', 'hy -m {modname}', "hy -c '(import {modname})'"])
+    'hy -m {modname}', "hy -c '(import {modname})'"])
```

Comment:
Something worth noting: `python foo.py` won't emit bytecode for `foo.py`. We not match that behaviour.

Reply:
I made `hy foo.hy` produce bytecode on purpose. The speed boost is a lot bigger for Hy than Python, so you shouldn't have to import from the file rather than running it directly in order to get bytecode.

Reply:
Noted, I'll revert that and add documentation for it.